changelog/Microsoft.IdentityGovernance.AccessReviews.json

```
@@ -1,55 +1,121 @@
-              "ChangedApiName": "accessReviewInstanceDecisionItemServicePrincipalResource",
-              "ChangeType": "Addition",
-              "Description": "Added the [accessReviewInstanceDecisionItemServicePrincipalResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemServicePrincipalResource?view=graph-rest-beta) resource type. This resource represents service principals whose access to a resource is represented through an [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstancedecisionitem?view=graph-rest-beta) object.",
-              "Target": "accessReviewInstanceDecisionItemServicePrincipalResource"
-            },
-            {
-              "Id": "c749c50d-8b7b-4274-ab46-45a49b9609c3",
-              "ApiChange": "Resource",
-              "ChangedApiName": "instance",
-              "ChangeType": "Addition",
-              "Description": "Added the **instance** relationship to [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItem?view=graph-rest-beta) resource.",
-              "Target": "accessReviewInstanceDecisionItem"
-            },
-            {
-              "Id": "c749c50d-8b7b-4274-ab46-45a49b9609c3",
-              "ApiChange": "Method",
-              "ChangedApiName": "recordAllDecisions",
-              "ChangeType": "Addition",
-              "Description": "Added the [recordAllDecisions](https://docs.microsoft.com/en-us/graph/api/recordAllDecisions?view=graph-rest-beta) method to the [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/accessreviewinstancedecisionitem?view=graph-rest-beta) resource.",
-              "Target": "recordAllDecisions"
-            }
-          ],
+      "ChangeList": [
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "accessReviewHistoryDecisionFilter",
+          "ChangeType": "Addition",
+          "Description": "Added the **accessReviewHistoryDecisionFilter** enumeration type.",
+          "Target": "accessReviewHistoryDecisionFilter"
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "accessReviewHistoryStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **accessReviewHistoryStatus** enumeration type.",
+          "Target": "accessReviewHistoryStatus"
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessReviewHistoryScheduleSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the [accessReviewHistoryScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryScheduleSettings?view=graph-rest-1.0) resource type.",
```

Recommending enhancing the descriptions for added context for the customer.
```suggestion
          "Description": "Added the [accessReviewHistoryScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryScheduleSettings?view=graph-rest-1.0) resource type. Use this resource to configure the settings of an [accessReviewHistoryDefinition](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewhistorydefinition?view=graph-rest-1.0) object.",
```